src/clients/ShieldAssociationClient.js

```diff
@@ -186,7 +186,7 @@ ShieldAssociationClient.prototype.getShieldAssociationsPerUser = function (usern
 
   var parameters = {
     options: {
-      url: this.baseUrl + '/shieldassociation/byuser/' + username,
+      url: this.baseUrl + '/shieldassociation/byUser/' + username,
```

Comment:
actually this is a pull request to only fix the test without my changes.  
and i just saw that the last commit in master failed in CI  :)